tests/admin/formatter/term-metabox-formatter-test.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Yoast\WP\Free\Tests\Admin\Formatter;
+
+use WPSEO_Term_Metabox_Formatter;
+use Yoast\WP\Free\Tests\Doubles\Admin\Formatter\Term_Metabox_Formatter_Double;
+use Yoast\WP\Free\Tests\TestCase;
+use Brain\Monkey;
+use Mockery;
+
+/**
+ * Unit Test Class.
+ *
+ * @coversDefaultClass WPSEO_Term_Metabox_Formatter
+ *
+ * @group Formatter
+ */
+class Term_Metabox_Formatter_Test extends TestCase {
+
+	/**
+	 * Holds the instance of the class being tested.
+	 *
+	 * @var Term_Metabox_Formatter_Double
+	 */
+	private $instance;
+
+	/**
+	 * Mocked stdClass.
+	 *
+	 * @var stdClass
+	 */
+	private $taxonomy;
+
+	/**
+	 * Mocked WP_Term.
+	 *
+	 * @var \WP_Term
+	 */
+	private $mock_term;
+
+	/**
+	 * Set up the class which will be tested.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->taxonomy                    = (object) array();
+		$this->mock_term                   = Mockery::mock( '\WP_Term' )->makePartial();
+		$this->mock_term->term_id          = 1;
```

Comment:
These alignments could use less pages (all the three lines)

Reply:
Done